frontend/components/Modal/modal.tsx

```diff
@@ -44,12 +44,21 @@ const Modal: React.FC<IModalProps> = ({
       {/* The backdrop, rendered as a fixed sibling to the panel container */}
       <div className="fixed inset-0 bg-black/30" aria-hidden="true" />
 
+      
+
       {/* This is the actual modal and it's contents */}
       <div className="fixed inset-0 overflow-y-auto">
         <div className="flex min-h-full items-center justify-center">
           <Dialog.Panel className="w-full max-w-5xl bg-white ">
             <div className="">
               <div className="px-4 py-3 md:px-12 md:py-10">
+              
+                <div className="flex justify-end sm:hidden">
+                  <button className="w-fit border font-bold bg-black border-black text-white rounded-full px-1.5" onClick={handleClose as any}>
```

Comment:
feel free to change button styles and forgive the `any`, wasn't sure correcty typing here.